WordPress/wporg-showcase-2022
# Register meta fields and add a panel to the editor to manage the content

Create a panel in the document sidebar to handle the custom meta fields: domain, country, author, and theme. These are plain text inputs, no validation is done (except for the default schema validation).

![editor](https://github.com/WordPress/wporg-showcase-2022/assets/541093/68b9b3b9-989e-4fd0-863a-4a457d39b6c7)

We can iterate on the fields, types, and help text.

Only domain is used currently, but the rest should save when edited. These will be used in the "more about" section, if they exist.

<img width="634" alt="Screenshot 2023-07-05 at 2 35 39 PM" src="https://github.com/WordPress/wporg-showcase-2022/assets/541093/efeeda0b-1203-47de-94bf-43d8ab0474ba">

**To test**
- Edit a site
- There may be values from the existing site data, these are the same meta keys
- Update some values
- Save the post, it should save
- Reload the page and/or check the domain on the frontend, it should be the content you just entered
